oeps/oep-0004.rst

```diff
@@ -121,6 +121,21 @@ Governance
 * Resources should not, on the other hand, be too broad to be useful, i.e.,
   instead of ``courseware``, consider ``discussions`` or ``catalog`` or
   whatever specific resource is of interest
+ 
+* Scope descriptions need to be presented to the user by the token management
+  system.  They should therefore be maintained with the central list of scopes.
+  Scope descriptions should be internationalized, but should be presented with
+  the scope name, which is consistent across translations.
+
+Alt:
+
+* Scope metadata should be maintained centrally with the canonical list of 
+  scopes.
+
+* Scope descriptions should be internationalized strings, managed by Transifex.
+
+* Scopes should be presented to the user during authorization as a combination 
+  of the canonical name and the internationalized description.
```

Comment:
I also prefer the alternate version. Keep an eye on this line in Governance, in the currently live version: `Centralize the complete list of scopes but decentralize the associated metadata and definitions`. You're proposing that we significantly alter this particular governance point with `Scope metadata should be maintained centrally...`. You should consider, in this PR, modifying that bullet - and consider whether there's a way to decentralize these descriptions or if centralizing them makes the most sense.